modules/search/class.jetpack-search.php

```diff
@@ -283,6 +291,36 @@ public function load_assets() {
 		}
 	}
 
+	/**
+	 * Registers a widget sidebar for Instant Search.
+	 */
+	public function register_jetpack_instant_sidebar() {
+		$args = array(
+			'name'          => 'Jetpack Instant Search Sidebar',
+			'id'            => 'jetpack-instant-search-sidebar',
+			'description'   => 'Customize the Jetpack Instant Search overlay sidebar',
```

Comment:
Quick addendum to [Greg's comment](https://github.com/Automattic/jetpack/pull/14283/files#r362023313): we should also change the description here to `Customize the Jetpack Search overlay sidebar`.